cfme/fixtures/utility_vm.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+import os.path
+
+import pytest
+
+from cfme.utils.conf import cfme_data
+from cfme.utils.conf import credentials
+from cfme.utils.generators import random_vm_name
+from cfme.utils.log import logger
+from cfme.utils.net import wait_pingable
+from cfme.utils.virtual_machines import deploy_template
+from cfme.utils.wait import TimedOutError
+
+
+@pytest.fixture(scope='module')
+def utility_vm():
+    """ Deploy an utility vm for tests to use.
+
+    This fixture creates a vm on provider and then receives its ip.
+    After the test run vm is deleted from provider.
+    """
+    try:
+        user_password = credentials['utility_vm']['password']
```

Comment:
I see this is called `root_password` in the fixture `network_share` down below. Should we keep it as `root_password` or leave it as `user_password`?

Reply:
I think this is fixed now. I defined injected_password in the creds and I am passing the whole thing to the consumers of this fixture